### ISSUE TITLE
Any chance to make a version compatible with bash

### ISSUE THREAD

**Issue by yhelioui** [makelinux/git-wizard#11]
Hello there,

I am very interesting with the git-wizard but would like to make it work on bash instead of zsh. I haven't yet checked what are the dependencies with zsh but do you think it could be feasible ?

Thanks,
Youssef

**Comment by makelinux** [comment]
Yes, it could be feasible. But why do you want to use bash? I've intentionally used zsh because it is more modern.

**Comment by yhelioui** [comment]
Hello,

Thanks a lot for your answer :)

I have nothing against zsh in particular but wanted to use the script within git bash without customizing the shell to use. I haven't found an easy way to install zsh on top of git bash.

I would be happy to contribute and make a version working on bash but would need guidance:
- Which part is requiring specifically zsh

Thanks,
Youssef

**Comment by yhelioui** [comment]
Hey,

Any chance you can point out the part that needs change ? I am really happy to contribute :)

Thanks again,
Youssef

**Comment by makelinux** [comment]
Try to run with `bash -x` and see errors.
